cmd/gnodev/test.go

```diff
@@ -291,7 +291,13 @@ func gnoTestPkg(
 			}
 
 			testFilePath := filepath.Join(pkgPath, testFileName)
-			err := tests.RunFileTest(rootDir, testFilePath, tests.WithSyncWanted(cfg.updateGoldenTests))
+			var fLogger func(args ...interface{}) = nil // note: tests.loggerFunc is not exported...
```

Comment:
Why not export `tests.loggerFunc`? It's an option anyways (`WithLoggerFunc`)

Reply:
By export do you mean changing `loggerFunc` -> `LoggerFunc` in the other package right?
As I'm still a new contributor I try to operate chirurgically but I admit what you say is more logical :)
Thanks

Reply:
Because you exported it now, you can use it as `var fLogger LoggerFunc` in `cmd/gnodev/test.go`